docs/root/intro/deprecated.rst

```diff
@@ -40,6 +59,20 @@ Deprecated items below are listed in chronological order.
     envoy.tracers.lightstep, envoy.lightstep
     envoy.tracers.zipkin, envoy.zipkin
 
+  .. note::
+    Some renamed filters produce metadata using their filter name as the metadata namespace:
+
+    * Mongo Proxy Filter
+    * Zookeeper Filter
+
+    The metadata generated by these filters may be consumed by the following extensions, whose
+    configurations may need to be adjusted to use the new names.
```

Comment:
Do you know if the protocol options are affected? I think they are also the extension name.

Reply:
Nope. Missed those. Will update again.

Reply:
So, AFAICT there's only thrift proxy and redis proxy with protocol options. No other network or http filter provides them (because no other network filters override `NetworkFilters::Common::FactoryBase::createProtocolOptionsTyped` and because no network or http filters override `ProtocolOptionsFactory::createProtocolOptionsConfig`).